identity/client/src/pages/users/detail/authorization/List.tsx

```diff
@@ -47,23 +112,150 @@ const List: FC<AuthorizationsListProps> = ({ user, loadingUser }) => {
 
   return (
     <>
-      <EntityList
-        title={t("Authorizations assigned to user")}
-        data={authorizations == null ? [] : authorizations.items}
-        headers={[{ header: t("ResourceType"), key: "resourceType" }]}
-        onEntityClick={showAuthorizationDetails}
-        addEntityLabel="Assign authorization"
-        onAddEntity={() => {}}
-        menuItems={[
-          {
-            label: t("Delete"),
-            onClick: () => {},
-            isDangerous: true,
-            icon: TrashCan,
-          },
-        ]}
-        loading={loadingAuthorizations}
-      />
+      <TableContainer
+        title="Authorizations"
+        description="Authorizations's of the user"
+      >
+        <DataTable<Authorization> rows={rows} headers={headers}>
+          {({
+            rows,
+            headers,
+            getHeaderProps,
+            getRowProps,
+            getTableProps,
+          }: DataTableRenderProps<Authorization, unknown[]>) => (
+            <div>
+              {/* Table Toolbar with Add Row Button */}
+              <TableToolbar>
+                <TableToolbarContent
+                  style={{
+                    display: "flex",
+                    justifyContent: "flex-end",
+                    gap: "16px",
+                  }}
```

Comment:
Please don't inline CSS. Identity uses `styled-components` so we should use that for styling.
Also the gap should use the Carbon [spacing tokens](https://carbondesignsystem.com/elements/spacing/overview/)